addons/project_issue/project_issue.py

```diff
@@ -68,37 +68,15 @@ class project_issue(osv.Model):
     }
 
     def _get_default_partner(self, cr, uid, context=None):
-        project_id = self._get_default_project_id(cr, uid, context)
-        if project_id:
-            project = self.pool.get('project.project').browse(cr, uid, project_id, context=context)
+        if context.get('default_project_id'):
```

Comment:
Check that context exists. Okay, you got the message, do it everywhere it is necessary :) .